### PR TITLE
Add get_param method to ParamGroupScheduler

### PR DESCRIPTION
**Pull request by sadra-barikbin**
In PR #2704 , in order to be able to have multiple groups in `FastaiLRFinder` when `step_mode != 'exp'`, I suggested this code:
https://github.com/pytorch/ignite/pull/2704/files#r975195661

which uses `ParamGroupScheduler`. `FastaiLRFinder` calls `get_param` on its scheduler in 
https://github.com/pytorch/ignite/blob/daa3c96d381f23b9900012f5defd6325abebbb63/ignite/handlers/lr_finder.py#L160

but `ParamGroupScheduler` has not this method. This PR adds that method.

### CODE REVIEW

File: ignite/handlers/param_scheduler.py [hunk 1]
```diff
@@ -1421,6 +1421,10 @@ def simulate_values(cls, num_events: int, schedulers: List[_LRScheduler], **kwar
 
             return values
 
+    def get_param(self) -> List[Union[float, List[float]]]:
+        """Method to get current `schedulers`' parameter values"""
```
Comment (vfdev-5) [inline ignite/handlers/param_scheduler.py:1425]:
```suggestion
        """
        Method to get current `schedulers`' parameter values
        
        .. versionadded:: 0.5.0
        """
```

**Review by vfdev-5 — APPROVED** [review]
Thanks for the PR @sadra-barikbin 
looks good to me.
Left a small comment on updating docstring

**Comment by vfdev-5** [comment]
I close and reopen the PR to restart the CI

**Comment by sadra-barikbin** [comment]
@vfdev-5 , `linkcheck` job failed due to `https://magellium.com` link in docs. What is it?

**Pull request merged**